src/Bybit/Spot/API/SymbolsInfo.jl

```diff
@@ -0,0 +1,98 @@
+module SymbolsInfo
```

Comment:
we use the singular for the names of the methods, so `SymbolsInfo ` it will be renamed `SymbolInfo `